lib/cocoapods/executable.rb

```diff
@@ -93,6 +94,21 @@ def self.popen3(bin, command, stdout, stderr)
       end
     end
 
+    def self.reader(input, output)
+      Thread.new do
+        buf = ''
+        begin
+          loop do
+            buf << input.readpartial(4096)
+            string, separator, buf = buf.rpartition(/[\r\n]/)
```

Comment:
It won't show up in a test of the return value, but rather in lines missing indentation in the indenter